### PR TITLE
Feat/DEVSU-2192-print-report-name-updates

### PR DESCRIPTION
**Pull request by bnguyen-bcgsc**
- DEVSU-2191
- DEVSU-2192
- DEVSU-2210
- Revise default file name for printed report (remove time zone, add 'report' after report name, and change 'probe' to 'targeted_gene', add server name)
- General code clean up for Genomic and Rapid report views

### CODE REVIEW

File: app/views/PrintView/index.tsx [hunk 1]
```diff
@@ -166,18 +166,29 @@ const Print = ({
...
+          case 'production':
+            serverName = '';
+            break;
+          case 'development':
+            serverName = '_iprdev';
+            break;
+          default:
+            serverName = '_iprstaging';
+            break;
+        }
```
Comment (elewis2) [inline app/views/PrintView/index.tsx:188]: maybe make 'staging' a case, and have the default value be null

File: app/views/ReportView/components/RapidSummary/index.tsx [hunk 2]
```diff
@@ -134,9 +132,9 @@ const RapidSummary = ({
   setIsLoading,
 }: RapidSummaryProps): JSX.Element => {
   const { report, setReport } = useContext(ReportContext);
-  const { isSigned, setIsSigned } = useContext(ConfirmContext);
+  const { setIsSigned } = useContext(ConfirmContext);
```
Comment (elewis2) [inline app/views/ReportView/components/RapidSummary/index.tsx:135]: this change isn't related to the filename is it?

**Review by elewis2 — CHANGES_REQUESTED** [review]

File: app/views/ReportView/components/RapidSummary/index.tsx (hunk 2 above)
Comment (bnguyen-bcgsc) [reply app/views/ReportView/components/RapidSummary/index.tsx:135]: no this one is just part of the code cleanup so eslint will stop throwing a fit over unused vars

File: app/views/ReportView/components/RapidSummary/index.tsx [hunk 3]
```diff
@@ -134,9 +132,9 @@ const RapidSummary = ({
...
   const { canEdit } = useReport();
-  const { showConfirmDialog } = useConfirmDialog();
+  // const { showConfirmDialog } = useConfirmDialog();
```
Comment (elewis2) [inline app/views/ReportView/components/RapidSummary/index.tsx:137]: to be removed?

**Review by elewis2 — APPROVED** [review]

**Pull request merged**